### ISSUE TITLE
Incompatible change in scratch org "created" property format in JSON output of "force:org:list"

### ISSUE THREAD

**Issue by SCWells72** [forcedotcom/cli#1091]
A customer reported this to me as it's breaking Illuminated Cloud's JSON-based integration with the CLI when calling `force:org:list`. It appears that in a very recent build, the format of the `created` property for `scratchOrg` entries has not only changed, but it's actually inconsistent even in the same list. For example, in one entry it's:

```
"created": "2021-07-24T03:30:41.000+0000"
```

and in another it's:

```
"created": "1626394643945"
```

e.g.,:

```
    "scratchOrgs": [
      {
        ...
        "created": "2021-07-24T03:30:41.000+0000",
        ...
      },
      {
        ...
        "created": "1626394643945",
        ...
      }
    ]
```

Traditionally it would have consistently been the latter.

I can certainly update Illuminated Cloud to accommodate for this, but since this is a breaking change to the effective API used by tools for integration, to me this seems like a regression that should be fixed ASAP.

**NOTE:** This doesn't seem to be happening consistently. I don't see it locally on any of my test machines on the latest CLI release (7.110.0-85b006b), all of which are binary installs. The user who reported this is using an NPM install on a Mac at the same version.

**NOTE 2:** Continuing to speak with the affected user, I'm wondering if perhaps it's not the CLI itself (at least not entirely) but perhaps something in his dev hub. He sees the same behavior after dropping back several CLI versions, and he's also seeing it in packaged install now in addition to the npm install. There's more discussion on the topic here if you're interested:

https://bitbucket.org/RoseSilverSoftware/illuminatedcloud/issues/1940

**Comment by attilah** [comment]
The root cause is that when a scratch org is created, in `scratchOrgInfoApi.ts` the `created` member had a value of `created: Date.now() + ''`, but in `sfdx-cli@latest-rc` this was changed to `created: scratchOrgInfoComplete.CreatedDate` which breaks integrations.

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-9651334](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000S6ODYA0/view)

**Comment by mshanemc** [comment]
this is fixed in the current RC 7.111.4+
